.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    # Monthly, 1st of the month, 6am
+    - cron: "0 6 1 * *"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
No Windows?

Reply:
Good catch.  I can always count on you to catch this.  😂 

I see that's what the appveyor workflow was doing.  Will add it.

Reply:
Windows workflow added, and everything passes.  🎉